src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -36,14 +37,15 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) =
 
     React.useEffect(() => {
         const aktivtVedtak = aktivVedtak(fagsak);
+        const httpMethod = visSubmitKnapp ? 'POST' : 'GET';
         if (aktivtVedtak) {
             axiosRequest<string, void>({
-                method: 'GET',
-                url: `/familie-ba-sak/api/dokument/vedtak-html/${aktivtVedtak?.id}`,
+                method: httpMethod,
+                url: `/familie-ba-sak/api/dokument/vedtaksbrev/${aktivtVedtak?.id}`,
             })
                 .then((response: Ressurs<string>) => {
                     if (response.status === RessursStatus.SUKSESS) {
-                        setBrev(response.data);
+                        setPdf(`data:application/pdf;base64,${response.data}`);
                         setErrorMessage(undefined);
                     } else if (response.status === RessursStatus.FEILET) {
                         setErrorMessage(response.melding);
```

Comment:
Use frontendFeilmelding? Do we need seperate error messages for pdf viewer and the rest? Why not just pass the error to PdfFrame?

Reply:
We had both HTML preview and PDF, so the error message was implemented in the upper level.
We can handle the error message in PdfFrame, but this makes PdfFrame no longer a pure viewer, and if the page gets more complicated, we still have to revert the change.
I think maybe just keep it here for now, but use the frontendFeilmelding instead. What do you think about it?